localisationExample/src/Particle.cpp

```diff
@@ -35,7 +40,7 @@ double Particle::_get_noise_sample_unfiorm(double minval, double maxval)
     return distribution(*_generatorPtr);
 }
 
-Pose Particle::getPosition()
+const Pose Particle::getPosition()
```

Comment:
When creating a new object, why make it const? It limits the user to reuse that variable if he wants.